.evergreen/config.yml

```
@@ -21,8 +21,23 @@ functions:
       params:
         script: |-
           set -o errexit
-          chmod u+x libmongocrypt/.evergreen/*.sh
-          ./libmongocrypt/.evergreen/print-env-info.sh
+          cd libmongocrypt
+          chmod u+x .evergreen/*.sh
+          ./.evergreen/print-env-info.sh
+
+          if [ "${is_patch}" != "true" ]; then
+            # determine if we have a release tag present on HEAD
+            head_tag=$(git tag -l --points-at HEAD '[0-9].*' | tail -1)
+          fi
+          echo "tag_upload_location: $head_tag"
+
+          cat <<EOT > ../tag_expansion.yml
+          tag_upload_location: "$head_tag"
+          EOT
```

TIL `EOT`. I am not sure if I prefer it to:
```suggestion
          echo 'tag_upload_location: "$head_tag"' >> ../tag_expansion.yml
```
Feel free to disregard this suggestion and resolve this comment as desired.

---

Good point.  This bit started as copy/pasta from another place where multiple lines are written to a file.  It turned out that I needed to handle this one its own in a different location and I didn't simplify the pattern.

---

However, I think that we actually need this:

```
echo "tag_upload_location: \"$head_tag\"" >> ../tag_expansion.yml
```

Otherwise with `'` we inhibit the expansion of `$head_tag`, which is a shell variable, not an Evergreen expansion.  I'll run a quick test in the other project to confirm the syntax.

---

It looks like you are correct. I was not aware `'` prohibits variable expansions.
Note, my suggestion above also uses `>>`; I do not know if that is appropriate either.